R/propose_snapshot.R

```diff
@@ -0,0 +1,55 @@
+#' @title Propose snapshot
+#' @export
+#' @family staging
+#' @description Propose a Production snapshot of Staging.
+#' @details [propose_snapshot()] proposes a snapshot of Staging
+#'   to migrate to Production. The recommended snapshot is the list of
+#'   packages for which (1) the build and check results of the current
+#'   release are in Staging, and (2) there are no issues.
+#' @return `NULL` (invisibly)
```

Comment:
Should document that the files `snapshot.json` and `snapshot.url` are written to 'path_staging' as a side effect.

Reply:
now documented